src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -253,16 +253,27 @@ public function notificationAction(Request $request)
         ));
     }
 
+    public function getPreferredModulesAction()
+    {
+        $controller = new \AdminModulesControllerCore();
+        ob_start();
+
+        $controller->ajaxProcessGetTabModulesList();
```

Comment:
@tchauviere I'm pretty sure we can re-use actions from the actual Controller, but don't know which one :)